.github/workflows/github-issues-monitor.yml

```diff
@@ -0,0 +1,30 @@
+name: GitHubIssues
+
+on:
+  issues:
+    types: [opened]
+
+jobs:
+  post-to-slack:
+    name: Post new issue to Slack channel
+    runs-on: ubuntu-latest
+    steps:
+    - name: Post to a Slack channel
+      uses: slackapi/slack-github-action@34c3fd73326693ef04728f8611669d918a2d781d # pin@v1.19.0
+      env:
+        SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN }}
+      with:
+        channel-id: '#github-new-issues'
+        payload: |
+          {
+            "text": "New Issue *${{ github.event.issue.number }}* opened by *${{ github.event.issue.user.login }}*",
+            "blocks": [
+              {
+                "type": "section",
+                "text": {
+                  "type": "mrkdwn",
+                  "text": "New Issue *${{ github.event.issue.number }}* opened by *${{ github.event.issue.user.login }}* \n Please assign the right team member to this issue and mark this Slack message with :white_check_mark: once done."
```

Comment:
Can we add a link to the GH issue here?

Reply:
Ah yes, that's what I wanted and I forgot :| I was just making sure the syntax is correct and it slipped my mind. Thanks @ebmifa